### PR TITLE
All: Translation: Update fr_FR.po

### PR DESCRIPTION
**Pull request by Mariem166**
some corrections to the old version

### CODE REVIEW

File: packages/tools/locales/fr_FR.po [hunk 1]
```diff
@@ -75,7 +78,7 @@ msgstr ""
 #: packages/app-desktop/plugins/GotoAnything.js:456
 #: packages/app-desktop/gui/KeymapConfig/utils/getLabel.js:20
 msgid "Goto Anything..."
-msgstr "Navigation rapide…"
+msgstr "aller n'importe ou... "
```
Comment (tessus) [inline packages/tools/locales/fr_FR.po:81]: Is there a reason why there's a space after `...`?
I've noticed this in many places, but I won't mark every single one of them.

Sometimes there are even 2 spaces with a sentence. I will mark the first occurrence.

File: packages/tools/locales/fr_FR.po [hunk 2]
```diff
@@ -251,7 +254,7 @@ msgstr "Barre latérale"
 #: packages/app-desktop/gui/MenuBar.js:614
 #: packages/app-desktop/gui/NoteList/commands/focusElementNoteList.js:18
 msgid "Focus"
-msgstr "Naviguer"
+msgstr "Concentrer "
```
Comment (tessus) [inline packages/tools/locales/fr_FR.po:257]: There's a trailing space in all of the next few tanslations.

File: packages/tools/locales/fr_FR.po [hunk 3]
```diff
@@ -388,7 +391,7 @@ msgstr "Révéler fichier dans dossier"
 
 #: packages/app-desktop/gui/NoteEditor/utils/contextMenu.js:89
 msgid "Copy path to clipboard"
-msgstr "Copier le chemin"
+msgstr "Copier le chemin dans  la presse-papiers"
```
Comment (tessus) [inline packages/tools/locales/fr_FR.po:394]: 2 spaces between `dans` and `la`

**Review by tessus — DISMISSED** [review]
Thanks for the update, but I have noticed 2 issues (I'm not really sure how that happened in the first place).

There are many occurrences of the following:

- a trailing space
- 2 spaces between 2 words

I've created 2 or 3 inline comments as an example what I mean. (But there are many more instances of these.)

Can you please fix those?

**Comment by laurent22** [comment]
There's a lot in there and I think the first step indeed would be to make sure that spaces are fixed. There are also a few mistakes like two commas in a row. Otherwise there's a mix of good changes and others that I don't quite agree with, but we can review this once the spaces and other small mistakes are fixed.

File: packages/tools/locales/fr_FR.po (hunk 3 above)
Comment (Mariem166) [reply packages/tools/locales/fr_FR.po:394]: I will fix it , thank you

**Review by Mariem166 — COMMENTED** [review]
fixedprevious errors

File: packages/tools/locales/fr_FR.po (hunk 3 above)
Comment (Mariem166) [reply packages/tools/locales/fr_FR.po:393]: done

**Comment by tessus** [comment]
Thanks for fixing the whitespace issues.

Laurent will have to comment on the French, since my French is not that great. Especially when it comes to what is used in applications and IT. I have only ever used English for OS and applications.

**Comment by laurent22** [comment]
I still see some small issues here and there, including double spaces, and the removal of spaces before colons (in French, there's a space before colons). Any chance these could be fixed?

**Comment by laurent22** [comment]
Scrolling down through the changes I see there are still spacing issues: for example, most spaces before ":", "?" or "!" have been removed, but in French there should be a space. I also still see double spaces and somehow spaces have also be added before "..." but there shouldn't be any.

**Comment by tessus** [comment]
@Mariem166 please let me know, if you plan on fixing the issues that Laurent mentioned. This PR has a conflict that has to be resolved as well, so this should be done first.

**Comment by tessus** [comment]
Closing for now. We can re-open this PR when needed.

**Pull request closed without merging**